angular/test/test-app/e2e/src/virtual-scroll.spec.ts

```diff
@@ -5,10 +5,7 @@ describe('Virtual Scroll', () => {
   })
 
   it('should open virtual-scroll', () => {
-    cy.document().then((doc) => {
-      const virtualElements = doc.querySelectorAll('ion-virtual-scroll > *');
-      expect(virtualElements.length).to.be.greaterThan(0);
-    });
+    cy.get('ion-virtual-scroll > *').its('length').should('be.gte', 0);
```

Comment:
```suggestion
    cy.get('ion-virtual-scroll > *').its('length').should('be.gt', 0);
```

This needs to be greater than instead of at least 0.